common/src/KokkosKernels_PrintConfiguration.hpp

```diff
@@ -18,11 +18,31 @@
 #define _KOKKOSKERNELS_PRINT_CONFIGURATION_HPP
 
 #include "KokkosKernels_config.h"
-
+#include "KokkosKernels_TplsVersion.hpp"
 #include <iostream>
 
 namespace KokkosKernels {
 namespace Impl {
+
+inline void print_cublas_version(std::ostream& os) {
```

Comment:
i would call this: `print_cublas_version_or_no_if_disabled`

Reply:
I went with `print_cublas_version_if_enabled` to keep the name's length in check :wink:

Reply:
I still think that`print_cublas_version_if_enabled` gives the idea that if enabled, print version but does not convey anything about what happens if not enabled. but since this might only be used for the print config, I think i can let it go